### PR TITLE
automate nuget package zipping and uploading

### PR DESCRIPTION
**Pull request by TyOverby**
@dotnet/roslyn-infrastructure @davkean

### CODE REVIEW

**Comment by TyOverby** [comment]
This tool has been used for the most recent zip-file package + upload.

**Comment by jmarolf** [comment]
:+1:

File: src/Tools/UploadNugetZip/CreateAndUpload.ps1 [hunk 1]
```diff
@@ -0,0 +1,60 @@
+function Get-ScriptDirectory
+{
+    $Invocation = (Get-Variable MyInvocation -Scope 1).Value
+    Split-Path $Invocation.MyCommand.Path
+}
+
+$ScriptDir = Get-ScriptDirectory
```
Comment (jaredpar) [inline src/Tools/UploadNugetZip/CreateAndUpload.ps1]: Could just use `$PSScriptRoot` here.

File: src/Tools/UploadNugetZip/CreateAndUpload.ps1 [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+$AzCopyLoc = "C:\Program Files (x86)\Microsoft SDKs\Azure\AzCopy\AzCopy.exe"
```
Comment (jaredpar) [inline src/Tools/UploadNugetZip/CreateAndUpload.ps1]:
```
Join-Path ${env:ProgramFiles(x86)} "Microsoft SDKs\Azure\AzCopy\AzCopy.exe
```

Accounts for the (corner) case of program files being on a non-C directory.

**Comment by jaredpar** [comment]
:+1:

File: src/Tools/UploadNugetZip/CreateAndUpload.ps1 (hunk 1 above)
Comment (TyOverby) [inline src/Tools/UploadNugetZip/CreateAndUpload.ps1]: Done

File: src/Tools/UploadNugetZip/CreateAndUpload.ps1 (hunk 2 above)
Comment (TyOverby) [inline src/Tools/UploadNugetZip/CreateAndUpload.ps1]: Done

**Pull request merged**